packages/dds/tree/api-report/tree.alpha.api.md

```diff
@@ -812,6 +813,14 @@ export interface SchemaFactoryObjectOptions<TCustomMetadata = unknown> extends N
     allowUnknownOptionalFields?: boolean;
 }
 
+// @alpha
+export enum SchemaFormatVersion {
+    // (undocumented)
```

Comment:
these members should probably have docs. V1 likley documenting that its supported in all clients which support shared tree (2.0 +)

and V2 (which should maybe be renamed to VUnstable) documenting that its format is not yet stable and thus should not be used for data which needs to be supported by future versions. When this code is actually all finished end to end, then it can be updated to V2, and have the version it will get released in noted.